doc/api.rst

```diff
@@ -164,7 +164,7 @@ You can add or prepend paths via the ``addPath()`` and ``prependPath()``
 methods::
 
     $loader->addPath($templateDir3);
-    $loader->prependPath($templateDir4);
+        ->prependPath($templateDir4);
```

Comment:
This is now invalid PHP code if the previous line terminates the statement with a semicolon.

Reply:
Furthermore, `addPath()` does not return the loader instance. So even when the semicolon on the line above is removed, this will not work as intended.